### ISSUE TITLE
Retri Pally dont have spells

### ISSUE THREAD

**Issue by DeathHandApproaching** [dalaranwow/dalaran-wow#5883]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Ive got to the 16th. There are no retri spells. Tab missing

**Expected behaviour**: Tab, spells, 1+2+3=dead

**Steps to reproduce the problem**:

1. Open spellbook
2. Dont see Retri tab
3. T_T

**Include proofs for this behaviour**

https://1drv.ms/u/s!Auov7zp7rGMpgYJWtr0L7TtuMhJSXw?e=Pt3t6L

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by DeathHandApproaching** [comment]
see ya

**Comment by Chenocide** [comment]
Did you make sure to learn any retribution abilities from the trainer? Otherwise you won't have a tab for retribution.
